### PR TITLE
wlfreerdp/input: Fix mouse wheel / touchpad scroll behavior

### PR DESCRIPTION
**Pull request by SaschaWessel**
Mouse wheel / touchpad scroll speed was too slow / too fast on Wayland.

This pr fixes the conversion of pointer axis scroll event values.

### CODE REVIEW

**Comment by hardening** [comment]
@freerdp-bot can you test please ?

File: client/Wayland/wlf_input.c [hunk 1]
```diff
@@ -152,13 +151,14 @@ BOOL wlf_handle_pointer_axis(freerdp* instance, const UwacPointerAxisEvent* ev)
 
 	input = instance->input;
 
-	direction = wl_fixed_to_int(ev->value);
+	direction = wl_fixed_to_double(ev->value) * 10.0;
```
Comment (akallabeth) [inline client/Wayland/wlf_input.c:154]: This is the only relevant change in the pr.
you just multiply the default value by 10.

Everything else down can be removed, as it does the same as the old code. (get an absolute value `step` and add it to the mask)

**Comment by akallabeth** [comment]
@SaschaWessel thank you for the pull request, it gave me the right idea where the bug was hidden.
I´ve created #6133 with a (smaller) fix. you got credits though ;)

File: client/Wayland/wlf_input.c (hunk 1 above)
Comment (SaschaWessel) [reply client/Wayland/wlf_input.c:154]: No, the code changes are required. Please see the following comparisson:

```
axis: 0 | value: -320 | direction/new:  255 | flags/new: 0x2ff | flags/old: 0x340
axis: 0 | value: -256 | direction/new:  255 | flags/new: 0x2ff | flags/old: 0x300
axis: 0 | value: -192 | direction/new:  192 | flags/new: 0x2c0 | flags/old: 0x2c0
axis: 0 | value: -128 | direction/new:  128 | flags/new: 0x280 | flags/old: 0x280
axis: 0 | value:  -64 | direction/new:   64 | flags/new: 0x240 | flags/old: 0x240
axis: 0 | value:    0 | direction/new:    0 | flags/new: 0x200 | flags/old: 0x200
axis: 0 | value:   64 | direction/new:  -64 | flags/new: 0x3c0 | flags/old: 0x340
axis: 0 | value:  128 | direction/new: -128 | flags/new: 0x380 | flags/old: 0x380
axis: 0 | value:  192 | direction/new: -192 | flags/new: 0x340 | flags/old: 0x3c0
axis: 0 | value:  256 | direction/new: -256 | flags/new: 0x300 | flags/old: 0x300
axis: 0 | value:  320 | direction/new: -256 | flags/new: 0x300 | flags/old: 0x340
```

**Comment by SaschaWessel** [comment]
> The pr looks good from the idea, you just use the wrong value range.
> `WheelRotationMask` gives you the maximum value allowed, which is larger than a `uint8_t`

No, there is one sign bit and 8 bits for the value.

**Pull request closed without merging**